src/scms/git.ts

```diff
@@ -1,46 +1,13 @@
-import fs from 'fs'
 import path from 'path'
 
-import findUp from 'find-up'
-import { Output, exec } from 'tinyexec'
```

Comment:
Why this line is changed by the way?

Reply:
`Output` is a type, so it's preferred to be first here. If you can help to change it with `type Output` that would be appreciated.

Reply:
Made this change in both files.